pkg/lib/registry/registry.go

```diff
@@ -122,7 +123,9 @@ func populate(ctx context.Context, loader registry.Load, graphLoader registry.Gr
 
 		if err = reg.Pull(ctx, ref); err != nil {
 			errs = append(errs, err)
-			continue
+			if !permissive {
+				continue
```

Comment:
Can we log a warning here?

Reply:
I did start like this, but then found that the pull error logs its own error, so thought it wasn't needed.